src/main/java/uk/gov/hmcts/cmc/claimstore/rules/CountyCourtJudgmentRule.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.claimstore.rules;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.time.LocalDate;
+
+@Service
+public class CountyCourtJudgmentRule {
+
+    public void assertCanIssueCountyCourtJudgment(final Claim claim) {
```

Comment:
Please do not use final modifier for arguments

Reply:
I think users request CCJ, justice system issues it. I would replace `issue` with `request`. Maybe `assertCountyCourtJudgementCanBeRequested`

Reply:
done

Reply:
We should also have some precondition to asses whether claim is not null as a precondition.

We could try `@NotNull` annotation so that IntelliJ warns us however that does not protect us from passing nulls at runtime.